.github/workflows/build.yml

```diff
@@ -28,8 +28,10 @@ jobs:
           python-version: "3.10"
       - name: Install dependencies
         run: pip install hatch
+      - name: Make required file path
+        run: mkdir -p dist
       - name: Run unit and integrations tests
-        run: hatch run test:pytest --cov=reqstool-python-decorators --cov-report=xml --cov-report=html 
+        run: hatch run test:pytest --cov=reqstool-client --cov-report=xml --cov-report=html -o junit_family=xunit2 --junitxml=dist/test_results.xml 
```

Comment:
What should we call the result file and where should we put it?

Reply:
See: https://github.com/Luftfartsverket/reqstool-client/blob/main/src/reqstool/requirements_indata/python/python_requirements_indata_paths.py#L15